src/vs/workbench/services/encryption/electron-browser/encryptionService.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@RMacfarlane why can this not move into `electron-sandbox` layer and be there for sandbox too?